.github/workflows/build-wheels.yml

```
@@ -0,0 +1,131 @@
+name: Build OFRAK Wheels
+
+# Trigger on push to feature/windows_tests branch
+on:
+  push:
+    branches:
+       - master
+#      - feature/windows_tests
+  workflow_dispatch:
+
+jobs:
+  build_wheels:
+    name: Build OFRAK wheels on ${{ matrix.os }} for Python ${{ matrix.python-version }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
+        
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          lfs: true
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18'
+
+      - name: Build frontend
+        run: |
+          if [ -d "frontend" ]; then
+            cd frontend
+            npm install
+            npm run build
+            cd ../ofrak_core
+            cp -r ../frontend/dist ofrak/gui/public
+          else
+            echo "No frontend directory found, creating empty public directory"
+            mkdir -p ofrak_core/ofrak/gui/public
+          fi
+        shell: bash
+
+      - name: Install build tools  
+        run: |
+          python -m pip install --upgrade pip
+          pip install cibuildwheel
```

```suggestion
          python3 -m pip install --upgrade pip
          python3 -m pip install cibuildwheel
```

Might be worth adding a `python3 --version` to visually confirm that it's running the right Python version here.